README.md

```diff
@@ -23,7 +23,7 @@ Here are some of the ways you can use Places:
 
 Places comprises the following components:
 
-* **Places web service** You can create and manage POIs by using the REST APIs. For more information about the REST APIs, see [Places web service](https://placesdocs.com/places-services-by-adobe-documentation/places-rest-apis).
+* **Places web service** You can create and manage POIs by using the REST APIs. For more information about the REST APIs, see [Places web service](places-rest-apis/README.md).
```

Comment:
- Please add a dash between the term and the text (Places web service - You can create...
- Please move the 2nd sentence down 1 line space and line up with the text in the bullet point, like this:

* **Places web service - 

  For more information...